01week/rockPaperScissors.js

```diff
@@ -7,27 +7,131 @@ const rl = readline.createInterface({
+*/
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  // First we need to sanitize the user input
+  // Use toLowerCase() method to change user input to lowercase so user isn't restricted by being case-sensitive
+  // After toLowerCase method, chain on the trim() method to trim whitespace
+  const hand1LowerTrim = hand1.toLowerCase().trim();
+  const hand2LowerTrim = hand2.toLowerCase().trim();
+
+  // boolean values to hold whether or not hand1 or hand2 is correct (defaults to null, will be set in switch statements for each hand)
+  let isHand1Correct = null;
+  let isHand2Correct = null;
+
+  // check to see if hand1 has correct inputs
+  switch(hand1LowerTrim) {
+    case "rock":
+    case "paper":
+    case "scissors":
+      isHand1Correct = true;
+      break;
+    default:
+      isHand1Correct = false;
+  }
+
+  // check to see if hand2 has correct inputs
+  switch(hand2LowerTrim) {
+    case "rock":
+    case "paper":
+    case "scissors":
+      isHand2Correct = true;
+      break;
+    default:
+      isHand2Correct = false;
+  }
+
+  // if both hand1 and hand2 have correct inputs
+  if (isHand1Correct && isHand2Correct) {
+
+    // code to test who wins goes here
+
+    // If (User1 AND User2 input is 'rock') OR (User1 AND User2 input is 'paper') OR (User1 AND User2 input is 'scissors'), it's a tie
+    if((hand1LowerTrim == "rock" && hand2LowerTrim == "rock") || (hand1LowerTrim == "paper" && hand2LowerTrim == "paper") || (hand1LowerTrim == "scissors" && hand2LowerTrim == "scissors")) {
+      return "It's a tie!";
+    }
+    // If User1 input is 'rock' AND User2 input is 'scissors', User1 wins
+    else if (hand1LowerTrim == "rock" && hand2LowerTrim == "scissors") {
+      return "Hand one wins!";
+    }
+    // If User1 input is 'rock' AND User2 input is 'paper', User2 wins
+    else if (hand1LowerTrim == "rock" && hand2LowerTrim == "paper") {
+      return "Hand two wins!";
+    }
+    // If User1 input is 'paper' AND User2 input is 'rock', User1 wins
+    else if (hand1LowerTrim == "paper" && hand2LowerTrim == "rock") {
+      return "Hand one wins!";
+    }
+    // If User1 input is 'paper' AND User2 input is 'scissors', User2 wins
+    else if (hand1LowerTrim == "paper" && hand2LowerTrim == "scissors") {
+      return "Hand two wins!";
+    }
+    // If User1 input is 'scissors' AND User2 input is 'paper', User1 wins
+    else if (hand1LowerTrim == "scissors" && hand2LowerTrim == "paper") {
+      return "Hand one wins!";
+    }
+    // If User1 input is 'scissors' AND User2 input is 'rock', User2 wins
+    else if (hand1LowerTrim == "scissors" && hand2LowerTrim == "rock") {
+      return "Hand two wins!";
+    }
+
+  }
+  // if hand1 and/or hand2 do NOT have correct inputs
+  else {
+
+    // if both hand1 and hand2 is NOT 'rock', 'paper', or 'scissors', print error message
+    if (isHand1Correct == false && isHand2Correct == false) {
```

Comment:
this can be simplified to 
```javascript
if(! isHand1Correct && !isHand2Correct) ...
```
Also you are repeating yourself A LOT here, what's a good way to simplify?